code/drasil-printers/lib/Language/Drasil/TeX/Helpers.hs

```diff
@@ -113,7 +113,7 @@ mkEnvArgSq nm args d =
 -- | Makes minipage environment.
 mkMinipage :: D -> D
 mkMinipage d = commandD "vspace" (command0 "baselineskip") $+$
-  command0 "noindent" $+$ mkEnvArgBr "minipage" "\\textwidth" d
+  command0 "noindent" $+$ mkEnvArgBr "minipage" "\\textwidth" d $+$ pure (text "")
```

Comment:
Could `empty` or something like it be used here?

Reply:
No, I tried that but `Text.PrettyPrint` tries to be smart and omits the empty line.